File: 10871.js
const fs = require("fs");
const rawInput = fs.readFileSync("input").toString().trim(); // fs.readFileSync(0, "utf-8").trim()
const input = rawInput.split("\n").map((row) => row.split(" ").map(Number));

const X = input[0][1];
const A = input[1];

console.log(A.filter((a) => a < X).join(" "));

File: algorithms/baekjoon/10871.py
# 제출 시 파일 입출력 부분 지우기

import sys

sys.stdin = open("input", "r")


def solution():
    X = int(input().split()[1])
    print(" ".join(filter(lambda numberString: int(numberString) < X, input().split())))


solution()
